src/Pages/LoginPage.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+import Stack from '@mui/material/Stack';
+import { PropTypes } from 'prop-types';
+
+function LoginPage({onClick}){
+    return(
+    <Stack spacing={2} style={{width: "25%", marginLeft: "35%"}}>
+      <TextField
+      id="username-id"
+      label="Username" 
+      variant="standard">
+     </TextField>
+     <TextField
+      id="password-id"
+      label="Password" 
+      type="password"
+      variant="standard">
+     </TextField>
+      <Button variant = "contained" id="login-button-id" onClick={onClick}>Login</Button>
```

Comment:
Read about `useHistory` hook. Redirect to another page using `useHistory` hook.

Reply:
useHistory is no longer exists in V6 of react-router, Introduced useNavigate.

Reply:
Ohh Nice!!